.yamato/package-documentation.yml

```diff
@@ -0,0 +1,15 @@
+{% metadata_file .yamato/metadata.metafile %}
+---
+
+generate_documentation:
+  name : Generate documentation
+  agent:
+    type: Unity::VM::osx
+    image: package-ci/macos-13:v4
```

Comment:
Do we sepcifically need a mac?

Reply:
Ubuntu image is also OK to run the brick, but based on my experience, Mac VM without GPU is more available than other typed of VMs, so it has a shorter waiting time.